test/unit/step-value-test.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const assert = require('assert');
```

Comment:
I've been using `chai expect` a bit and have been liking it. It can auto generate errors on failed asserts for something like: `expect(false).to.be.ok`